projects/plugins/jetpack/modules/theme-tools/site-logo/inc/class-site-logo.php

```diff
@@ -109,29 +106,49 @@ public function customize_register( $wp_customize ) {
 			'site_logo',
 			array(
 				'capability'        => 'manage_options',
-				'default'           => array(
-					'id'    => 0,
-					'sizes' => array(),
-					'url'   => false,
-				),
+				'default'           => 0,
 				'sanitize_callback' => array( $this, 'sanitize_logo_setting' ),
 				'transport'         => 'postMessage',
 				'type'              => 'option',
 			)
 		);
 
+		$size_settings = jetpack_get_site_logo_dimensions();
+		$width         = isset( $size_settings['width'] ) ? $size_settings['width'] : null;
+		$height        = isset( $size_settings['height'] ) ? $size_settings['height'] : null;
+
 		// Add our image uploader.
 		$wp_customize->add_control(
-			new Site_Logo_Image_Control(
+			new WP_Customize_Cropped_Image_Control(
```

Comment:
Since we're now using the same format for `site_logo` as Core does for the custom_logo theme mod, we can use the same control in the Customizer from Core, and delete the code for the old control.